gui/default/syncthing/device/idqrModalView.html

```diff
@@ -1,7 +1,11 @@
 <modal id="idqr" status="info" icon="fas fa-qrcode" heading="{{'Device Identification' | translate}} - {{deviceName(currentDevice)}}" large="yes" closeable="yes">
-  <div class="modal-body">
-    <div class="well well-sm text-monospace text-center" select-on-click>{{currentDevice.deviceID}}</div>
-    <img ng-if="currentDevice.deviceID" class="center-block img-thumbnail" ng-src="qr/?text={{currentDevice.deviceID}}" alt="qr code" />
+  <div class="modal-body text-center">
+    <div class="well well-sm text-monospace" select-on-click>{{currentDevice.deviceID}}</div>
+    <div>
+      <a href="qr/?text={{currentDevice.deviceID}}" target="_blank" download="{{currentDevice.deviceID}}.png">
```

Comment:
Maybe better to use the short ID? I find files with such long "gibberish" names a bit ugly.